dlt/common/destination/capabilities.py

```diff
@@ -53,6 +54,9 @@ class DestinationCapabilitiesContext(ContainerInjectableContext):
     insert_values_writer_type: str = "default"
     supports_multiple_statements: bool = True
     supports_clone_table: bool = False
+    scd2_high_timestamp: pendulum.DateTime = HIGH_TS
```

Comment:
clickhouse max timestamp is 2099, so I needed to make this configurable for each destinations, I'm not satisfied that the capabilities are the right place, if you have a better idea for it, lmk

Reply:
we need the same timestamp in each destination. my question to @jorritsandbrink why not to allow nulls? so the active record is marked as having no end date

Reply:
this is how I did it in my initial implementation, that would make it a bit easier